teams/team-spock/dev/values.yaml

```diff
@@ -0,0 +1,12 @@
+clusterA:
+  clusterName: 
+clusterB:
+  clusterName:   
+common:
+  provider-config-aws-name: 
```

Comment:
Lets not hard code or keep these variables empty this in a GitOps based approach, then is not fully automated. lets pass it via argo to GitOps. Same with region var.

Reply:
yes done